### DISCUSSION TITLE
Complete box art collection for Flycast 2.2

### DISCUSSION THREAD

**Discussion by HungryWolf79** [flyinghead/flycast#1358] [Q&A]
Hello Guys,
Do any of you know how to implement or download the complete box art cover collection for Flycast 2.2?
For example f355 or animal basket or psyvariar 2 and many others have no box art image at all....
Thank you!!

**Comment by flyinghead** [comment]
Flycast downloads game information and box art from https://thegamesdb.net/. You can create a gamesdb account and add missing games.

**Comment by HungryWolf79** [comment]
Thank you for the answer. I did the account now. How i can add games? Opening flycast online or i have to download the missing files from somewhere?

**Comment by flyinghead** [comment]
Use their web site to add games: https://thegamesdb.net/add_game.php
Select platform 'Arcade' for arcade games and 'Sega Dreamcast' for console games.

**Comment by HungryWolf79** [comment]
Thank you again. Ok i 'll try it. I have to put just the name of the game in the request field or it is better to put also the rom name? example: 
  psyvariar 2 or psyvar2.zip??
I tried your link but it is not working.  But i have found in the site where to do the request.
